Gir_Gio.toml

```diff
@@ -0,0 +1,30 @@
+[options]
+girs_dir = "gir-files"
+library = "Gio"
+version = "2.0"
+min_cfg_version = "2.32"
+target_path = "."
+work_mode = "normal"
+
+generate = [
+    "Gio.Action",
+    "Gio.ActionGroup",
+    "Gio.ActionMap",
+    "Gio.AppInfo",
+    "Gio.AppLaunchContext",
+    "Gio.Application",
+    "Gio.Menu",
+    "Gio.MenuAttributeIter",
+    "Gio.MenuItem",
+    "Gio.MenuLinkIter",
+    "Gio.MenuModel",
+    "Gio.Permission",
+    "Gio.Settings",
+    "Gio.SimpleAction",
+    "Gio.SimpleActionGroup",
+    "Gio.SimplePermission",
+]
```

Comment:
At least they don't seem to require `gio-unix` so we could try them too. There are some thread safety issues though:

> GVolumeMonitor is not [thread-default-context aware](https://developer.gnome.org/glib/stable/glib-The-Main-Event-Loop.html#g-main-context-push-thread-default), and so should not be used other than from the main thread, with no thread-default-context active.

We don't have a strategy for dealing with that outside `gdk`/`gtk`. Could consider lifting the main_thread stuff to `glib` but here the limitations are documented in a cryptic way :(